frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_appearance/direction_setting.dart

```diff
@@ -141,3 +141,29 @@ class TextDirectionSetting extends StatelessWidget {
     }
   }
 }
+
+class EnableRTLToolbarItemsSetting extends StatelessWidget {
+  const EnableRTLToolbarItemsSetting({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return ThemeSettingEntryTemplateWidget(
+      label: LocaleKeys.settings_appearance_enableRTLToolbarItems.tr(),
+      trailing: [
+        Switch(
+          key: const ValueKey('enable_rtl_toolbar_items_switch'),
```

Comment:
define the string as const value

Reply:
and mark it as onlyvisiblefortesting

Reply:
Done the visible for testing part. But the string was const before I didn't get which string you meant.